### DISCUSSION TITLE
weblate db shema

### DISCUSSION THREAD

**Discussion by Husam600** [WeblateOrg/weblate#9539] [Q&A]
Hi, I have a question I want to use Weblate with an existing db in my project but after checking the config for weblate for db below:
POSTGRES_PASSWORD = weblate
POSTGRES_USER = weblate
POSTGRES_DATABASE = weblate
POSTGRES_HOST = Database2
POSTGRES_PORT =

It looks like Weblate needs to create it's own database with weblate name the question is it possible to create another db schema in my existing db and add it for weblate ,if yes where can i add postgres shema name in the configuration

**Comment by nijel** [comment]
Just change  `POSTGRES_DATABASE = weblate` to something else.

**Reply by Husam600** [reply]
Yes I do this then weblate will try to use the my public schema and I use the public schema for other project So I want to add my db schema somewhere in config

**Reply by nijel** [reply]
Django currently only supports changing schema using custom configuration, see https://code.djangoproject.com/ticket/28774
